include/zephyr/drivers/adc.h

```diff
@@ -402,44 +402,6 @@ struct adc_dt_spec {
  */
 #define ADC_DT_SPEC_INST_GET(inst) ADC_DT_SPEC_GET(DT_DRV_INST(inst))
 
-/**
```

Comment:
note: has been moved because the dt-spec version requires calling a function defined below.